pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "chart-operator"
 	source      = "https://github.com/giantswarm/chart-operator"
-	version     = "1.0.0-dev"
+	version     = "1.0.0"
```

Comment:
I will rollback it after pushing tags.

Reply:
Thanks. Minor but we should set the version to `1.0.1-dev` so it isn't a rollback.

Reply:
Yes, I will upgrade it to 1.0.1-dev after tags.